src/effects/effectchainslot.h

```diff
@@ -108,7 +108,7 @@ class EffectChainSlot : public QObject {
     void slotControlClear(double v);
     void slotControlChainEnabled(double v);
     void slotControlChainMix(double v);
-    void slotControlChainSuperParameter(double v, bool force = false);
+    void slotControlChainSuperParameter(double v, bool force);
```

Comment:
Why did you decide keep the default parameter shouldEmit = true in slotChainEffectChanged() although you pass `true` explicitly now? Default parameters in slots should be avoided.